server/core/store.go

```diff
@@ -554,6 +554,15 @@ func (s *StoresInfo) GetMetaStores() []*metapb.Store {
 	return stores
 }
 
+// RemoveTombStoneRecords remove all the record of tombstone Records.
+func (s *StoresInfo) RemoveTombStoneRecords() {
+	for storeID, store := range s.stores {
+		if store.IsTombstone() {
+			delete(s.stores, storeID)
```

Comment:
Only remove in the memory, restarting PD will appear again.

Reply:
Fixed , Thanks !